com.unity.cinemachine/Documentation~/CmCamera.md

```diff
@@ -1,4 +1,4 @@
-# CmCamera
+# CmCameras
```

Comment:
This should not be plural.  It's a class name, and we never pluralize them this way